### ISSUE TITLE
Cannot convert from "FString" to "FName"

### ISSUE THREAD

**Issue by HtnrxUNc** [EsotericSoftware/spine-runtimes#2204]
Hello, sir
I tried to follow the instructions and have created the "Plugins" directory in the project's directory and put the "SpinePlugin" directory under "Plugins" and "spine-cpp" under "Plugins\SpinePlugin\Source\SpinePlugin\Public" directory.
I have installed "visual studio 2015 update 3"
When I tried to run the project using UE, he prompted me that I need to build, which I agreed to, but after that he reported an error.

`Demo could not be compiled.Try rebuilding from source manually.`

I tried to find the "Log.txt" in the "UE_4.21\Engine\Programs\UnrealBuildTool" directory
This is the complete file, I don't know why there are non-English characters in this file even after I changed all the languages of UE and windows to English, but I tried to translate it to English using a translation device

[Log.txt](https://github.com/EsotericSoftware/spine-runtimes/files/10088438/Log.txt)

If you check some path errors, it may be because I tried to modify the path output in the log, if his does not have a very big impact, then please ignore it.

My windows version: Windows Feature Experience Pack 120.2212.4170.0
My Unreal Engine version: 4.21.2
The project I'm pulling from is pulled using git, and I'm guessing I'm pulling from 
8fb3876396d52d0535905b18d57f1c89e6845f46

Do I need to provide any other information, please let me know if so.

**Comment by HtnrxUNc** [comment]
I am using visual studio version: visual_studio_community_2015_with_update_3_x86_x64

**Comment by HtnrxUNc** [comment]
Hello, I forgot the output after translation

[Log.txt](https://github.com/EsotericSoftware/spine-runtimes/files/10088485/Log.txt)

**Comment by badlogic** [comment]
I'm terribly sorry, I entirely overlooked this issue!

Is there any specific reason you are using Unreal Engine 4.21.2 and Visual Studio 2015? I'd strongly recommend updating to at least Unreal Engine 4.26 and Visual Studio 2017.

**Comment by HtnrxUNc** [comment]
> I'm terribly sorry, I entirely overlooked this issue!
> 
> Is there any specific reason you are using Unreal Engine 4.21.2 and Visual Studio 2015? I'd strongly recommend updating to at least Unreal Engine 4.26 and Visual Studio 2017.

Hi, I'm very sorry for the late reply.
I can't test if it's a version issue at the moment, so I won't be opening this issue again at the moment to cause others problems.
The reason I used this version is because at the time I installed it I didn't check the relevant version requirements on the project page, so I used google to search for other people's processes. Others suggested this version for me to use.
I will try the version you requested again afterwards if I can to check if the problem will happen again.
Thanks again for your patience and answers.
